ode/implicit.tex

```diff
@@ -422,11 +425,14 @@ \section{A- and B-stability}
   for all $h$.
 \end{Theorem}
 
-% \begin{remark}
-%   Der Begriff A-stabil wurde von Dahlquist bewusst neutral
-%   gewählt. Insbesondere steht also A-Stabilität
-%   \underline{\textbf{nicht}} für asymptotische Stabilität.
-% \end{remark}
+ \begin{remark}
+	The term of A-stability was deliberately chosen neutrally by Dahlquist
+	since he considered notions with a rather extravagant choice of words as outworn.
+	Note that A-stability does not stand for asymptotic stability. The notion
+	of B-stability which we will consider later on was introduced by John
```

Comment:
Das kann ich in der Vorlesung sagen, aber es kann nicht im Skript stehen :-)

Reply:
In Ordnung, ist geändert :-)